modules/map/relative_map/relative_map.cc

```diff
@@ -289,6 +289,15 @@ void RelativeMap::LoadNavigationPath() {
     priority++;
   }
 
+  // Generate the hash code, dreamview does not update the navigation data
+  // having the same hash code with the latest navigation data.
+  // Generate hash code before filling proto header.
+  std::string proto_content;
+  navigation_info.SerializeToString(&proto_content);
+  static std::hash<std::string> hash_func;
+  std::size_t hash_code = hash_func(proto_content);
+  navigation_info.set_hashcode(hash_code);
```

Comment:
https://github.com/ApolloAuto/apollo/blob/master/modules/common/util/message_util.h#L92

Reply:
The hashcode is generated before setting header field, so it's just generated based on the topic's content, or path info.
This pull request is going to merge to branch v3.1_dev, and there is no file message_util.h in branch v3.1_dev, so hashcode generating function is declared locally.

Reply:
@siyangy This pull request tries to prevent topics being updated with the same path info, and timestamps could distinguish topics rather than the content of topics. If we publish the two topics with the same path info on different time, both of them will be updated once.